core/string/unpack/1_spec.rb

```diff
@@ -0,0 +1,12 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+
+ruby_version_is "2.4" do
+  describe "String#unpack1" do
+    it "returns the first value of #unpack" do
+      "ABCD".unpack1('x3C').should == "ABCD".unpack('x3C')[0]
+      "\u{3042 3044 3046}".unpack1("U*").should == 0x3042
+      "aG9nZWZ1Z2E=".unpack1("m").should == "hogefuga"
+      "A".unpack1("B*").should == "01000001"
```

Comment:
This look like it fails on MRI 2.4.0 on Darwin:
```
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-darwin15]                                                                                         
1)
String#unpack1 returns the first value of #unpack FAILED
Expected "\x00\x00\x00\x00\x00\x00\x00\x00"
 to equal "01000001"
/Users/travis/build/ruby/spec/core/string/unpack/1_spec.rb:9:in `block (3 levels) in <top (required)>'
/Users/travis/build/ruby/spec/core/string/unpack/1_spec.rb:4:in `block in <top (required)>'
/Users/travis/build/ruby/spec/core/string/unpack/1_spec.rb:3:in `<top (required)>'
```
That sounds like a bug, maybe fixed in 2.4.1 or 2.4.2.

Reply:
I updated to 2.4.2 on Darwin, could you rebase on latest master?